docs/user-guide/configuring-containers.md

```diff
@@ -93,21 +93,16 @@ pods/hello-world
 
 ## Validating configuration
 
-If you’re not sure you specified the resource correctly, you can ask `kubectl` to validate it for you:
-
-```console
-$ kubectl create -f ./hello-world.yaml --validate
-```
+We enable validation by default in `kubectl` now.
```

Comment:
nit: Something about the wording on the line feels a little clunky. This could be reworded to something like "As of $COMMIT/$RELEASE configuration validation is done by default. [...] and no longer requires a special switch"